api.go

```diff
@@ -281,7 +281,7 @@ func (r *Router) Call(ctx context.Context, bucketID uint64, mode CallMode,
 
 	for {
 		if spent := time.Since(requestStartTime); spent > timeout {
-			r.metrics().RequestDuration(spent, fnc, false, false)
+			r.metrics().RequestDuration(spent, fnc, "", false, false)
```

Comment:
We discussed privately that every request to any replicaset should generate a metric. So, I suggest these fixes to the current PR: #72